src/common/config.go

```diff
@@ -114,8 +111,13 @@ service:
           - .metadata.annotations."opslevel.com/ignore"
 `
 
-func GetConfig(data string) (*Config, error) {
+func ParseConfig(data string) (*Config, error) {
```

Comment:
`Get` with argument `string` implies there's something inside the string that we are fetching, since we're create an object based on the contents of the string `Parse` is more accurate.